app/assets/javascripts/components/shared/tooltip.js

```diff
@@ -0,0 +1,64 @@
+(function (App) {
+
+  App.Component.Tooltip = Backbone.View.extend({
```

Comment:
Can you  add a quick doc somewhere to say this component should be extended?
If I don't know it, I could try to directly instantiate this component and it would throw an error because `this.content` doesn't exist.